### ISSUE TITLE
yaru cursors: kde/plasma support

### ISSUE THREAD

**Issue by jbouter** [ubuntu/yaru#2921]
**Expected Behavior**

For all cursors to properly display, in a KDE plasma behavior

**Actual Behavior**

Key components and symbolic links are missing that KDE plasma expects

**Steps to Reproduce the Problem**

1. Use KDE Plasma
2. Install the yaru cursor theme
3. Enable the cursor theme
4. Observe missing icons, falling back to the breeze cursors

**Yaru version**

```
20.04.11.1
```

**Missing Icons**

Thus far, I've been able to identify the following missing symbolic links:

missing file | should link to
------------|----------
pointing_hand | pointer
whats_this | help
half-busy | progress
ibeam | text
split_v | row-resize
split_h | col-resize
size-ver | default
size-hor | default
size-fdiag | default
forbidden | no-drop
closedhand | dnd-move
wait | watch

The following files are present in the breeze cursor directory, but not in Yaru, and I have not been able to identify which files they should point to yet:

```
center_ptr
color-picker
down-arrow
draft
grab
left-arrow
openhand
right-arrow
up-arrow
wayland-cursor
x-cursor
```

I hope it's possible for these symbolic links to be created, in order to be able to fully use the Yaru cursor theme within KDE Plasma.

**Comment by Muqtxdir** [comment]
Hi @jbouter 👋
Thanks for sharing the files that needed to be linked. I'll do this

**Comment by jbouter** [comment]
Hi @Muqtxdir 👋🏻 

Thanks! If there's any assisting I can do, please let me know. :-)

**Comment by Muqtxdir** [comment]
Hi @jbouter 👋

I've created this PR: #2958 for all the **missing-symlinks** (_still need to do wayland-cursor and x-cursor_), please tryout this PR when free and share your experience

**Comment by jbouter** [comment]
Hi @Muqtxdir ! Brilliant, I'll give it a go as soon as I can. I am however missing `grab` and `openhand` from the list. They seem to be related to each other.

How can I go about building your PR? I haven't really worked with cursors before, and only ever downloaded "compiled" cursors. Can I download the content of your PR and move it to the proper directory, or do I need to build cursors prior to doing so?

**Comment by Muqtxdir** [comment]
> How can I go about building your PR? I haven't really worked with cursors before, and only ever downloaded "compiled" cursors. Can I download the content of your PR and move it to the proper directory, or do I need to build cursors prior to doing so?

Sharing the instructions from [CONTRIBUTING.md](https://github.com/ubuntu/yaru/blob/master/CONTRIBUTING.md)
**Step1:**
```bash
# Cloning Yaru from GitHub and switching to PR branch:

git clone https://github.com/ubuntu/yaru
cd yaru
git switch plasma-symlinks-to-cursors 
```
**Step2:**
```bash
# Installing Yaru  

meson build
cd build
sudo ninja install
```

**Comment by jbouter** [comment]
Ah, I could've found that myself. Sorry! Got it compiled and working though, and it looks absolutely lovely. I also really like the darker "loading" cursors that replace the blue circle.

Anyway, all the cursors that I could visually discover missing are now working. I'll test it for a few more days, but it sure looks like the newly added symlinks resolve the issue.

**Comment by jbouter** [comment]
Hi @Muqtxdir! 👋🏻 

After using the cursor theme for a while, I've found no inconsistencies anymore. Even though there seem to be a few symlinks missing, I have no idea where those cursors would appear as I've not been able to spot them. As far as I am concerned, your PR resolves the issue! :-)

**Comment by Muqtxdir** [comment]
Hi @jbouter,

Thanks for sharing the feedback in the time you used the PR-branch and for alsp pointing out that there's a couple of symlinks missing that need to be added (*I've made a list of the missing-ones comparing it to the awesome Breeze cursor-set and will update the PR*). I'll look into the symlinks once #2944 is finished and merged into `master` to avoid any conflict. 

*Note: also, sorry if takes a while for #2958 to land in `master` based on how #2944 progresses*

**Comment by jbouter** [comment]
Hey @Muqtxdir,

You're most welcome! Thank you for taking this feature request serious - considering Yaru isn't necessarily a KDE project, you wouldn't have been wrong to dismiss the request to begin with.

I looked at #2944 earlier and I'm pretty amazed with how that's turning out. Obviously, this request will rely on #2944's progression. That's entirely fine of course, I'm just happy to have this resolved for any other KDE/Plasma users that wish to use Yaru cursors instead (and to be honest, if it lands before 22.04 hits - we should be good right?)

**Comment by Muqtxdir** [comment]
Thanks @jbouter you've been really helpful with the missing symlinks list on this issue 👍
